assignments/closure.js

```diff
@@ -1,14 +1,34 @@
 // ==== Challenge 1: Write your own closure ====
 // Write a simple closure of your own creation.  Keep it simple!
-
+hello = "Hello"
+function Greet(param){
```

Comment:
Conventionally we reserve capital letters for Classes and Constructor functions in JS